### PR TITLE
feat: add front end for JoyIDInfo

### PR DESCRIPTION
**Pull request by jwrookie**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
+      version: 'detect',
+    },
+    'import/resolver': {
+      node: {
+        paths: ['src'],
+        extensions: ['.js'],
+      },
+    },
+  },
+  rules: {
```
Comment (duanyytop) [inline .eslintrc.js]: Why close prettier ?

Comment (jwrookie) [reply .eslintrc.js]: 配置了eslint，但没有遵守它的规则，ide中报红比较恶心，因此关闭了它

Comment (duanyytop) [reply .eslintrc.js]: Please follow the rules of eslint and prettier

File: example/metadata.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ const run = async () => {
     pubKey: "0x650e48cf029c8a04788c02d7d88bad7b62918714137d0cd486b5b3aff53d0c2baecabd8d23107933f85fdf13cd814a0ba3d1848329b0504d7134a88962e9bde3",
     credentialId: "0x459d12c09a65e58e22a9d8d6fa843c3d",
     alg: "0x01",
+    frontEnd: "https://twitter.com",
```
Comment (duanyytop) [inline example/metadata.ts]: Maybe "https:://app.joy.id" is more better.

File: .eslintrc.js (hunk 1 above)
Comment (jwrookie) [reply .eslintrc.js]: ok, changed

File: example/metadata.ts (hunk 2 above)
Comment (jwrookie) [reply example/metadata.ts]: yes, changed

**Review by duanyytop — APPROVED** [review]

**Pull request merged**